unsplash/photo.go

```diff
@@ -38,13 +38,23 @@ type ExifData struct {
 	Iso          *int    `json:"iso"`
 }
 
+// Tag lists can be applied to any photo
+type Tag struct {
+	Type  *string `json:"type"`
+	Title *string `json:"title"`
+}
+
 // Photo represents a photo on unsplash.com
 type Photo struct {
 	ID           *string    `json:"id"`
 	CreatedAt    *time.Time `json:"created_at"`
+	UpdatedAt    *time.Time `json:"updated_at"`
 	Width        *int       `json:"width"`
 	Height       *int       `json:"height"`
 	Color        *string    `json:"color"`
+	Description  *string    `json:"description"`
+	AltText      *string    `json:"alt_description"`
+	Views        *int       `json:"views"`
```

Comment:
Where is Views and alt_description property documented?
I don't see it in https://unsplash.com/documentation#get-a-photo.

Reply:
Indeed, these appear to be undocumented.

I simply found them in the api responses.

Reply:
Let me verify these on my end.
I'm less concerned about these and more about the tags.
There is no documentation around tags and I want to figure out if it is a free form key value pair or is it always going to have `title` and `type` as the key.

Reply:
I can verify everything but `tags`. I don't see `tags` in any photos. I tried 50 random photos but hit none with `tags`.

Reply:
I should have thought to provide some examples.

This photo has tags, at least for me. 
https://api.unsplash.com/photos/U1iYwZ8Dx7k

I am seeing two types of tag.  `search` has only a title (and type), but `landing_page` has a large structure under `source`.

Reply:
This is confusing but it checks out. Thank you for providing a link for my sanity.